docs/source/release-history.rst

```diff
@@ -2,6 +2,17 @@
 Release History
 ***************
 
+v1.15.2 (2020-06-12)
+====================
+
+Added
+-----
+
+* Various documentation additions.
+* `jsonschema 2.x` compatibility.
```

Comment:
ReST, unlike Markdown, needs two back-ticks for code formatting. One back-ticket gives italics. :shrug: